### PR TITLE
Content re-organization

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: layouts/partials/navbar.html [hunk 1]
```diff
@@ -45,11 +46,31 @@
...
             <a class="navbar-item" href="{{ .URL }}">
               {{ .Title }}
             </a>
-            {{- end }} <!-- range $docs -->
+            {{ end }} <!-- range $docs -->
+          </div>
+        </div>
+
+        <div class="navbar-item has-dropdown is-hoverable">
+          <a class="navbar-link" href="/docs">
```
Comment (Hoverbear) [inline layouts/partials/navbar.html]:
```suggestion
          <a class="navbar-link" href="/deep-dive/">
```

File: layouts/partials/navbar.html [hunk 2]
```diff
@@ -45,11 +46,31 @@
           </a>
 
           <div class="navbar-dropdown is-hidden-touch">
-            {{- range $docs }}
+            {{ range $docs }}
             <a class="navbar-item" href="{{ .URL }}">
               {{ .Title }}
             </a>
-            {{- end }} <!-- range $docs -->
+            {{ end }} <!-- range $docs -->
```
Comment (Hoverbear) [inline layouts/partials/navbar.html]: Right now the `Docs` ion Navbar links to `/docs/` (a blank page), can it go to `/docs/concepts/overview/`?

Comment (lucperkins) [reply layouts/partials/navbar.html]: Good catch

**Comment by lucperkins** [comment]
@Hoverbear I think I've addressed your concerns. I've also added a redirect for `/docs -> /docs/concepts/overview` in case there are any stray links out there.

**Pull request merged**